### PR TITLE
Add sc-component manager

### PR DESCRIPTION
**Pull request by MksmOrlov**
Add sc-component manager
use sc-component branch from https://github.com/ostis-ai/sc-component-manager/pull/67

### CODE REVIEW

File: README.md
Comment (NikitaZotov) [inline README.md]:
```suggestion
```

File: README.md
Comment (NikitaZotov) [inline README.md]:
```suggestion
```

File: README.md
Comment (NikitaZotov) [inline README.md]:
```suggestion
  ./run_sc_component_manager.sh
```

File: scripts/install_platform_with_component_manager.sh
Comment (NikitaZotov) [inline scripts/install_platform_with_component_manager.sh:21]: Please, add ci for check this script

File: scripts/set_vars.sh
Comment (NikitaZotov) [inline scripts/set_vars.sh:27]: Where ims.ostis.kb venvs were provided?

File: scripts/install_submodules.sh
Comment (NikitaZotov) [inline scripts/install_submodules.sh:49]: Add, please, warning for installation of ims.ostis.kb

File: scripts/build_sc_component_manager.sh
Comment (NikitaZotov) [inline scripts/build_sc_component_manager.sh:10]: Please, comment this setups to prevent removing by misunderstanding

**Review by NikitaZotov — CHANGES_REQUESTED** [review]

File: README.md
Comment (MksmOrlov) [reply README.md]: why?

File: scripts/install_platform_with_component_manager.sh
Comment (MksmOrlov) [reply scripts/install_platform_with_component_manager.sh:21]: ok, I will add it after merging https://github.com/ostis-ai/sc-component-manager/pull/67

File: scripts/install_submodules.sh
Comment (MksmOrlov) [reply scripts/install_submodules.sh:49]: There is no alternative, that merged now

Comment (NikitaZotov) [reply scripts/install_submodules.sh:49]: Why? But you add sc-component-manager

File: README.md
Comment (NikitaZotov) [inline README.md]: Why you need to pass this config? It passed in venvs

File: scripts/install_platform.sh
Comment (NikitaZotov) [inline scripts/install_platform.sh]: Is it not script to install platform without manager of components?

**Review by NikitaZotov — CHANGES_REQUESTED** [review]

File: scripts/install_platform_with_component_manager.sh
Comment (NikitaZotov) [reply scripts/install_platform_with_component_manager.sh:21]: Let's merge this PR after marging PR https://github.com/ostis-ai/sc-component-manager/pull/67

File: scripts/install_submodules.sh
Comment (MksmOrlov) [reply scripts/install_submodules.sh:49]: sc-component-manager is not deliting ims.ostis.kb. You can install it. The warning should be added after merging ostis-standard specification

File: scripts/install_platform.sh
Comment (MksmOrlov) [reply scripts/install_platform.sh]: It is is not is it, yes

File: README.md
Comment (MksmOrlov) [reply README.md]: image

Comment (NikitaZotov) [reply README.md]: By default, sc-server provide CONFIG_PATH from venvs. Can we make this for sc-component-manager?

**Review by NikitaZotov — APPROVED** [review]

**Review by MakarenkoAI — APPROVED** [review]

**Pull request merged**
